### DISCUSSION TITLE
Run in Supervisord

### DISCUSSION THREAD

**Discussion by 5kr1p7** [spatie/laravel-short-schedule#49] [Q&A]
Command definition (`app/Console/Commands/SkudProcess.php`):
```php
...
class SkudProcess extends Command
{
    protected $signature = 'skud:process';

    public function handle()
    {
        Log::debug('Shortrun tick.');
        ...

        return 0;
    }
}
```

`app/Console/Kernel.php` file:
```php
namespace App\Console;

use App\Console\Commands\SkudProcess;
use Illuminate\Console\Scheduling\Schedule;
use Illuminate\Foundation\Console\Kernel as ConsoleKernel;
use Spatie\ShortSchedule\ShortSchedule;

class Kernel extends ConsoleKernel
{
     protected $commands = [
        //
    ];

    protected function shortSchedule(ShortSchedule $shortSchedule)
    {
        $shortSchedule->command('skud:process')->everySeconds(5)->withoutOverlapping()->onOneServer();
    }

    protected function commands()
    {
        $this->load(__DIR__ . '/Commands');

        require base_path('routes/console.php');
    }
}

```

Supervisord config looks like:
```ini
[program:short-schedule]
directory=/var/www/html
command=php /var/www/html/artisan short-schedule:run --lifetime=300 --no-interaction -vvv
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0
autostart=true
autorestart=true
startretries=0
```

I don't see my debug messages `Shortrun tick.`. Now I using `command=/bin/sh -c "while [ true ]; do (php /var/www/html/artisan skud:process -vvv --no-interaction); sleep 5; done` in supervisord.

How I can fix this?

P.S. I have latest version of package.

**Comment by Wiaimer** [comment]
Supervisord config:

[program:short-schedule]
directory=/var/www/html
command=php artisan short-schedule:run --lifetime=300 --no-interaction -vvv
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
stderr_logfile=/dev/stderr
stderr_logfile_maxbytes=0
autostart=true
autorestart=true
startretries=0

Because directory is already defined, command does not require full paths,i'm going to add the full path to command,also, no Debug message is displayed.you can try do not add full paths to command
